src/status_im/multiaccounts/key_storage/core.cljs

```diff
@@ -109,12 +109,15 @@
                    types/json->clj
                    :error
                    string/blank?
-                   not)]
+                   not)
+        onboarding? (not (or (:multiaccounts/login db) (:multiaccount db)))]
```

Comment:
Hi @bitgamma,
Can you please tell me what `:multiaccounts/login` represents?

If it represents that the user is logging in(not logged yet) then shouldn't logic be?

`onboarding? (or (:multiaccounts/login db) (not (:multiaccount db))))`

Reply:
:multiaccounts/login is the account selected for login, while :multiaccount is the logged in :multiaccount. If neither is set, I am onboarding, which is what I want to check here

Reply:
Oh great 👍